h/static/scripts/directive/test/util.js

```diff
@@ -1,13 +1,40 @@
 'use strict';
 
-// converts a camelCase name into hyphenated ('camel-case') form,
-// as Angular does when mapping directive names to tag names in HTML
+/**
+ * Converts a camelCase name into hyphenated ('camel-case') form.
+ *
+ * This matches how Angular maps directive names to HTML tag names.
+ */
 function hyphenate(name) {
   var uppercasePattern = /([A-Z])/g;
   return name.replace(uppercasePattern, '-$1').toLowerCase();
 }
 
 /**
+ * Helper for retrieving an Angular module in a test.
+ *
+ * Given the 'inject' function from the 'angular-mocks' module,
+ * retrieves an instance of the specified Angular module.
+ */
+function ngModule(inject, name) {
```

Comment:
It will be called as `util.ngModule` where `var util = require('./util')` so I hope it should be obvious enough that it isn't part of angular. All the angular builtins are accessed via `angular.<foo>`.